### ISSUE TITLE
reduce_mean 함수 동작 원리

### ISSUE THREAD

**Issue by 1core2life** [KHU-Dcom-Study/2018-1-DeepLearning-Study#24]
accuracy 확인 시 tf.reduce_mean 사용하는데, 원리가 무엇 인가요?

**Comment by 1core2life** [comment]
accuracy를 구하는 방식은 다음과 같다. 생성된 model 의 prediction 값과 라벨 값을 비교하여, tf.equal 함수를 이용한다. 그리고 이 데이터들에 대해 reduce_mean을 구하는 것 이다. 이때, reduce_mean은 단순히 평균값을 구하는 것 뿐이다.
